core/modules/replica/RemoveReplicaQservMgtRequest.cc

```diff
@@ -84,11 +79,14 @@ RemoveReplicaQservMgtRequest::RemoveReplicaQservMgtRequest(
         _qservRequest(nullptr) {
 }
 
-std::string RemoveReplicaQservMgtRequest::extendedPersistentState(SqlGeneratorPtr const& gen) const {
-    return gen->sqlPackValues(id(),
-                              databases(),
-                              chunk(),
-                              force() ? 1 : 0);
+std::map<std::string,std::string> RemoveReplicaQservMgtRequest::extendedPersistentState() const {
+    std::map<std::string,std::string> result;
+    for (auto&& database: databases()) {
+        result["database"] = database;
```

Comment:
Only last database is stored in a map. Do you maybe want something like comma-separated list of databases?